servers/src/main/java/tachyon/master/file/journal/InodeDirectoryEntry.java

```diff
@@ -35,8 +35,10 @@ public InodeDirectoryEntry(@JsonProperty("creationTimeMs") long creationTimeMs,
       @JsonProperty("pinned") boolean pinned,
       @JsonProperty("lastModificationTimeMs") long lastModificationTimeMs,
       @JsonProperty("childrenIds") Set<Long> childrenIds,
-      @JsonProperty("permissionStatus") PermissionStatus ps) {
-    super(creationTimeMs, id, name, parentId, persisted, pinned, lastModificationTimeMs, ps);
+      @JsonProperty("username") String username, @JsonProperty("groupname") String groupname,
```

Comment:
I see, in that case when I merge my PR into the master, the authentication branch should make sure to specify one property per line to limit the scope of future merge conflicts